.rubocop_todo.yml

```diff
@@ -7,12 +7,12 @@
 
 # Offense count: 8
 Metrics/AbcSize:
-  Max: 51
+  Max: 53
```

Comment:
As features and their correctness are superior to style pedantry, I want to push #160.

I think the reason that people just stab RuboCop offenses is that it runs *before* RSpec. It prevents maintainers to just see the feature implemented correctly or not.